### DISCUSSION TITLE
Ability to parse a "packet" without a CCSDS header

### DISCUSSION THREAD

**Discussion by joeymukherjee** [CCSDSPy/ccsdspy#124] [Ideas]
I'd like to parse a large amount of bits defined similarly to a regular CCSDS packet, but without a CCSDS header, and would like an API to do that.  

So, why not make a fake CCSDS header?  In my case, it is because the number of bytes is greater than 65535 and would require the continuation packet and all that.  

In theory, all the decoding code is there, and should be just a matter of having an API to do it.  Would it be possible?

Thanks!
Joey

**Comment by ddasilva** [comment]
Do you have any thoughts on this @ehsteve ?
